config.js

```diff
@@ -17,10 +17,11 @@ mongo_hacker_config = {
   sort_keys:      false,            // sort the keys in documents when displayed
   uuid_type:      'default',        // 'java', 'c#', 'python' or 'default'
   banner_message: 'Mongo-Hacker ',  // banner message
-  version:        '0.0.9',          // current mongo-hacker version
+  version:        '0.0.10',         // current mongo-hacker version
   show_banner:     true,            // show mongo-hacker version banner on startup
   windows_warning: true,            // show warning banner for windows
   force_color:     false,           // force color highlighting for Windows users
+  count_deltas:    true,            // "count documents" shows deltas with previous counts
```

Comment:
Done in 5814bed _(although I agree: this probably wouldn't be an issue for very many use cases)_